webplugin/js/app/zendesk-chat-service.js

```diff
@@ -144,12 +145,14 @@ function ZendeskChatService() {
                     if (currentMessageDetail.to === userId) {
                         username = "User";
                     } else {
-                        username = currentGroupData.displayName;
+                        //Get bot name
+                        var botId = currentMessageDetail.to;
+                        username = _this.getBotNameById(botId, userDetails);
```

Comment:
botId variable can be removed and directly the value can be used

Reply:
Ok👍

Reply:
Done.